templates/experiments/single_experiment_home.html

```diff
@@ -335,47 +490,7 @@ <h3 class="font-bold text-lg">Parent Experiments:
     }
 
     document.addEventListener('DOMContentLoaded', () => {
-      new TomSelect('#tag-multiselect-filter', {
-        plugins: ["remove_button", "caret_position", "input_autogrow"],
-        maxItems: null,
-        hidePlaceholder: true,
-      });
-
-      new TomSelect('#participants-filter', {
-        plugins: ["remove_button", "caret_position", "input_autogrow"],
-        valueField: 'identifier',
-        searchField: ['name', 'identifier'],
-        hidePlaceholder: true,
-        load: function(query, callback) {
-          const url = '{% url "participants:search" request.team.slug %}?q=' + encodeURIComponent(query);
-          fetch(url)
-            .then(response => response.json())
-            .then(json => {
-              callback(json.results);
-            }).catch(()=>{
-              callback();
-            });
-        },
-        render: {
-          option: function(item, escape) {
-            return `<div class="">
-                      <span class="h4">
-                        ${ escape(item.name || 'No Name') }
-                      </span>
-                      <span class="text-muted"> (${ escape(item.identifier) })</span>
-                    </div>`;
-          },
-          item: function(item, escape) {
-            return `<div class="">
-                      <span class="h4">
-                        ${ escape(item.name || 'No Name') }
-                      </span>
-                      <span class="text-muted"> (${ escape(item.identifier) })</span>
-                    </div>`;
-          }
-        },
-      });
-
+      showTabContent('allsessions');
```

Comment:
Why did this need to be added? Also it won't work if the user doesn't have 'view_chat' permissions

Reply:
Hmm, I think somewhere the tab is being hidden unexpectedly rather. It should be showing up by default, since the `allsessions` tab is checked by default